### PR TITLE
Reverts #73206 and #73214

### PR DESCRIPTION
**Pull request by optimumtact**
let tim cook

### CODE REVIEW

**Comment by MrStonedOne** [comment]
thats not even the right pr thats fix pr #2

**Comment by MrStonedOne** [comment]
https://github.com/tgstation/tgstation/pull/73206

**Comment by optimumtact** [comment]
LET ME COOK

**Comment by Timberpoes** [comment]
image

**Comment by ZephyrTFA** [comment]
what are we cooking

File: code/controllers/subsystem/admin_verbs.dm [hunk 1]
```diff
@@ -72,11 +72,20 @@ GENERAL_PROTECT_DATUM(/datum/controller/subsystem/admin_verbs)
 			cached_formats[verb_module] = verb_module_formatted
 
 		var/original_name = verb_information[VERB_MAP_NAME]
+		if(!cached_formats[original_name])
```
Comment (Mothblocks) [inline code/controllers/subsystem/admin_verbs.dm:75]: this whole file shouldnt exist anymore, how'd it get in here

**Comment by Mothblocks** [comment]
image

**Pull request closed without merging**